internal/controller/cloud-control/vpcpeering_aws_test.go

```diff
@@ -19,6 +19,7 @@ var _ = Describe("Feature: KCP VpcPeering", func() {
 			remoteVpcId     = "6e6d1748-9912-4957-9075-b97a6fac8ac1"
 			remoteAccountId = "123"
 			remoteRegion    = "eu-west1"
+			connectionId    = "26ce833e-07d1-4493-98ee-f9d6f11a6987->6e6d1748-9912-4957-9075-b97a6fac8ac1"
```

Comment:
Is AWS peering connection id really in this format?

Reply:
No. It is actually something like pcx-111aaa111 but this was a quick way for me to create unique identifier between two vpcs.